disco_aws_automation/disco_vpc_gateways.py

```diff
@@ -273,3 +277,15 @@ def destroy_nat_gateways(self):
         # Need to wait for all the NAT gateways to be deleted
         wait_for_state_boto3(self.boto3_ec2.describe_nat_gateways, filter_params,
                              'NatGateways', 'deleted', 'State')
+
+        #Release EIPs of auto-
+        eips = {
+            gw['SubnetId']: gw['NatGatewayAddresses'][0]['PublicIp']
+            for gw in gateways['NatGateways']
+        }
+        for network in self.disco_vpc.networks.values():
+            nat_gateways = self.disco_vpc.get_config("{0}_nat_gateways".format(network.name))
```

Comment:
OOps who ever is deleting VPC might not have access to config it was created with.